packages/terra-form/tests/nightwatch/select-field/select-field-spec.js

```diff
@@ -0,0 +1,40 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-form', done);
+  },
+
+  'Displays a default SelectField with a single option': (browser) => {
```

Comment:
Maybe break these tests up a bit. The test 'displays a pouplated..' looks like its testing multiple things: that it was present, that it was rendered correctly, as well tests the select functionality.